.travis.yml

```diff
@@ -7,6 +7,9 @@ before_install:
   - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
   - sudo apt-get update
   - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+  # the docker user and password are defined in
+  # https://app.travis-ci.com/github/resonai/ybt/settings
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
why is this login needed?

Reply:
docker-hub is getting more and more aggressive about users creating account and choose a plan so they started limiting the non-logged-in pulls and we are getting:
```
Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```
see https://app.travis-ci.com/github/resonai/ybt/jobs/554916402